### ISSUE TITLE
Json schemas references included in a library are not validated properly

### ISSUE THREAD

**Issue by nmarinel01** [raml-org/raml-js-parser-2#603]
Having this example, that includes a library usage with an include of an external json schema, parser fails with the following error: 

![image](https://cloud.githubusercontent.com/assets/19980188/21809367/26f5c1d2-d726-11e6-8785-3259f041ee01.png)

[Archive.zip](https://github.com/raml-org/raml-js-parser-2/files/696304/Archive.zip)

**Comment by ddenisenko** [comment]
Which parser version do you use? I can not reproduce with the latest parser.

**Comment by nmarinel01** [comment]
Hi @ddenisenko !
we are using release 1.1.12
Regards !

**Comment by ddenisenko** [comment]
I checked this sample in API Workbench and with parser 1.1.12/1.1.13, and it works.

I suspect this can be caused by the interaction between the parser and the file system resolver you use. Could you log, which files are requested by the parser from file system resolver and whether resolver answers with the proper content in each case?

**Comment by agustinlg** [comment]
The problem comes after expansion, you need to call the `expand(true)` and then call the `toJson({rootNodeDetails: true})`.

**Comment by ddenisenko** [comment]
Thank you, @agustinlg , it is reproducible with that calls.

I get the following error:

```
"errors": [
    {
      "code": "INHERITING_UNKNOWN_TYPE",
      "message": "Inheriting from unknown type",
      "path": "api.raml",
      "range": {
        "start": {
          "line": 8,
          "column": 8,
          "position": 127
        },
        "end": {
          "line": 8,
          "column": 12,
          "position": 131
        }
      }
    }
  ]
```

**Comment by KonstantinSviridov** [comment]
Fixed by the commit:
https://github.com/raml-org/raml-js-parser-2/commit/6699eeb19ec87c1114b28516c58f334c74d5004f

**Comment by KonstantinSviridov** [comment]
Thanks for the report, @nmarinel01 

The issue will be fixed in the next parser release (1.1.14)

Regards, Konstantin
